src/main/java/com/cburch/logisim/data/TestVector.java

```diff
@@ -131,7 +131,11 @@ private void parseHeader() throws IOException {
             throw new IOException("Test Vector header format error: bad spec: " + t);
 
           columnName[i] = t.substring(0, s);
-          int w = new Integer(t.substring(s + 1, e)).intValue();
```

Comment:
@BFH-ktt1: Maybe I am wrong, but isn't `Integer`'s value range limited to 32 bits? It seems like this code is not yet adjusted to the maximum bit width of 64 bits.